.opt-prof.yml

```diff
@@ -0,0 +1,77 @@
+# Runs OptProf for MSBuild binaries based on an insertion build
+
+trigger: none # Prevents this pipeline from triggering on check-ins
+pr: none # don't run this on PR as well, this should only trigger off on MSBuild runs
+
+resources:
+  pipelines:
+  - pipeline: ComponentBuildUnderTest
+    source: MSBuild\MSBuild
```

Comment:
Had to look this up, in case it helps other reviewers it's `Name of the pipeline that produces the artifact`